spigot.yml.md

```diff
@@ -0,0 +1,670 @@
+
+* Domyślną wartością jest **You are not whitelisted on this server!**.
+
+### unknown-command
+Opcja ta odpowiada za wiadomość którą zobaczy gracz gdy wpisze komendę która nie jest dostępna na serwerze.
+
+* Domyślną wartością jest **Unknown command. Type "help" for help.**.
+
+### server-full
+Opcja ta odpowiada za wiadomość którą zobaczy gracz próbujący zalogować się na serwer gdy ten jest pełny.
+
+* Domyślną wartością jest **The server is full!**.
+
+### outdated-client
+Opcja ta odpowiada za wiadomość którą zobaczy gracz próbujący wejść na serwer, ale ma starszą wersję gry niż serwer obsługuje.
+
+* Napis `{0}` w tekście zostanie automatycznie zamieniony na wersję gry obsługiwaną przez serwer.
+* Domyślną wartością jest **Outdated client! Please use {0}**.
+
+### outdated-server
+Opcja ta odpowiada za wiadomość którą zobaczy gracz próbujący wejść na serwer, ale ma nowszą wersję gry niż serwer obsługuje.
+
+* Napis `{0}` w tekście zostanie automatycznie zamieniony na wersję gry obsługiwaną przez serwer.
+* Domyślną wartością jest **Outdated server! I'm still on {0}**.
+
+### restart
+Opcja ta odpowiada za wiadomość którą zobaczą wszyscy gracze online gdy serwer będzie restartowany poleceniem `/restart`.
+
+* Domyślną wartością jest **Server is restarting**.
+
+## Stats
+Sekcja ta odpowiada za ustawienia dotyczące statystyk i ich przechowywania przez serwer.
+
+### disable-saving
+Opcja ta odpowiada za ustawienie czy serwer powinien zapisywać statystyki graczy i ich osiągnięcia (przed minecraftem 1.12).
+
+* Wpisz `false` żeby statystyki były zapisywane.
+* Wpisz `true` żeby statystyki nie były zapisywane.
+
+### forced-stats
+Opcja ta odpowiada za ustawienie które statystyki i osiągnięcia (przed minecraftem 1.12) gracz powinien mieć wymuszone przez serwer.
+
+* Wpisz `{}` żeby nie wymuszać żadnych statystyk.
+* Wpisz 
+    ```yml
+    forced-stats:
+    achievement.openInventory: 1
+    ```
+    aby ustawić osiągnięcie `open inventory` jako od razu zaliczone dla każdego gracza.
+* Domyślną wartością jest **{}**.
+
+## Advancements
+Sekcja ta odpowiada za ustawienia dotyczące postępów i ich przechowywania przez serwer.
+
+### disable-saving
+Opcja ta odpowiada za ustawienie czy serwer powinien zapisywać postępy graczy.
+
+* Wpisz `false` żeby postępy nie były zapisywane.
+* Wpisz `true` żeby postępy były zapisywane.
+
+### disabled
+Opcja ta odpowiada za ustawienie które postępy powinny zostać wyłączone na serwerze. Wyłączone postępy nie będą się wyświetlały w menu postępów. Jeżeli zablokujesz postęp wymagany przez inne postępy, które nie będą zablokowane, w konsoli serwera pojawią się błędy i postępy mogą nie działać poprawnie. Pamiętaj żeby blokować wszystkie postępy zależne od innych blokowanych postępów.
+
+* Wpisz `[]` żeby nie wyłączać żadnych postępów.
+* Wpisz to, co znajduje się w poniższym spoilerze, żeby zablokować wszystkie standardowe postępy.
```

Comment:
Wystarczy

```yml
- '*'
```

:D

Reply:
artykuł na stronie spigotowej mówił co innego, ale skoro faktycznie tak jest to skoryguję ;)